server/app/views/applicant/ProgramCardsSectionFragment.html

```diff
@@ -1,6 +1,6 @@
 <!doctype html>
 
-<div
+<section
```

Comment:
This is to augment the semantics of the document. A normal section has a `generic` aria-role, but when a section has `aria-labelledby` browsers automatically set the role to `region`. This will aid assistive technologies and allows me to select the these elements in the same way for browser tests.